lib/cli.js

```diff
@@ -107,31 +121,34 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' The --require and -r flags are deprecated. Requirements should be configured in package.json - see documentation.');
 	}
 
+	// copy resultant cli.flags into conf for use with Api and elsewhere
+	conf = Object.assign(conf, cli.flags);
```

Comment:
`Object.assign(conf, …)` modifies and returns `conf`. There's no need to reassign it (since you're reassigning it to itself).